python/common/org/python/types/List.java

```diff
@@ -418,7 +418,13 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
                 }
                 return new org.python.types.List(sliced);
             } else {
-                int idx = (int) ((org.python.types.Int) index).value;
+                int idx;
+                if (index instanceof org.python.types.Bool){
+                    idx = ((org.python.types.Bool)index).value ? 1 : 0;
```

Comment:
This seems the case of doing:

    idx = ((org.python.types.Bool) index).__int__().value;